Assets/Scripts/Managers/HouseManager.cs

```diff
@@ -25,6 +25,9 @@ private void Start()
 
         Debug.Log("HouseRating: "+houseValue);
 		SetHouseMode(HouseMode.Explore);
+
+		audioManager = FindObjectsOfType<AudioManager>();
+        audioManager[0].Play("Building");
```

Comment:
As previously mentioned. Same applies to the other managers too.
```suggestion
		audioManager = FindObjectOfType<AudioManager>();
		audioManager.Play("Building");
```